runtime/src/bank.rs

```diff
@@ -966,7 +966,7 @@ pub(super) enum RewardInterval {
 }
 
 impl Bank {
-    fn wrap_with_bank_forks_for_tests(self) -> (Arc<Self>, Arc<RwLock<BankForks>>) {
+    pub fn wrap_with_bank_forks_for_tests(self) -> (Arc<Self>, Arc<RwLock<BankForks>>) {
```

Comment:
This PR seems like a good opportunity to move these public constructors/functions under DCOU. It'll be good cleanup for bank.rs.

Reply:
Let's use https://github.com/solana-labs/solana/blob/e1165aaf00575cb2860cfa128d6ecd57bbf0b4eb/accounts-db/src/accounts_db.rs#L9549 as an example, where it has a separate impl block for all DCOU functions.

Reply:
Here in `bank.rs`, I can include this attribute in all functions whose name end with `for_tests`.

Reply:
How about define a new `impl Bank {}` and move all these test functions there?

Reply:
Yes, that's going to be cleaner.